airbyte-proxy/nginx-auth.conf.template

```diff
@@ -37,6 +37,23 @@ http {
         auth_basic off;
       }
     }
+
+    location /api/v1/workspaces {
```

Comment:
Should this be `location ~ ^/api/v1/workspaces/.*` instead.

Reply:
This doesn't need to be a regex, from their [docs](https://docs.nginx.com/nginx/admin-guide/web-server/web-server/#locations)
> To find the location that best matches a URI, NGINX Plus first compares the URI to the locations with a prefix string. It then searches the locations with a regular expression.

> Higher priority is given to regular expressions, unless the ^~ modifier is used. Among the prefix strings NGINX Plus selects the most specific one (that is, the longest and most complete string).

This non-regex version will still match `/api/v1/workspaces/list`, `/api/v1/workspaces/get`, `/api/v1/workspaces/*`.